examples/js/test-js.js

```diff
@@ -87,6 +87,18 @@ const TestJs = () => (
     )}
   >
     <Slide>
+      <Slide>
+        <Markdown>{`
+        # hello there
+        
+        | hi   | Supported | Versions |
+|-----------|-----------|----------|
```

Comment:
This indentation looks strange.

Reply:
Yeah it's wrong, it's just not immediately noticeable that it's wrong 😭

Reply:
For this specific case, I don't see how we're going to handle it outside of something super complicated like probablistically calculating indent based on markdown parsing.

The only idea here I can come up with is "indent cannot be `0`" and ignore a detected min indent that is `0` relying on everything else....